lib/msal-common/src/app/config/MsalModuleConfiguration.ts

```diff
@@ -63,6 +63,11 @@ const DEFAULT_NETWORK_OPTIONS: INetworkModule = {
 };
 
 const DEFAULT_CRYPTO_IMPLEMENTATION: ICrypto = {
+    createNewGuid: (): string => {
+        const notImplErr = "Crypto interface - createNewGuid() has not been implemented";
+        console.warn(notImplErr);
```

Comment:
any reason to use warn here if we throw an error?

Reply:
also why not use logger? Maybe this is leftover from developing

Reply:
Logger will be implemented in a separate PR, I am using console for now to make things a little easier. I will replace the console warnings with logger errors.